Apps/Common/test/unit/Auditing/DbAuditLoggerTests.cs

```diff
@@ -240,5 +234,51 @@ public async Task ShouldWriteAuditEventHandleException()
                     It.IsAny<Func<It.IsAnyType, Exception?, string>>()),
                 Times.Once);
         }
+
+        private static GetPopulateWithHttpContextMock SetupGetPopulateWithHttpContextMock(bool useRouteValues, bool useQueryParamValues)
```

Comment:
Nit: I think the type should be named `PopulateWithHttpContextMock` and the method name should be either `SetupPopulateWithHttpContextMock` or `GetPopulateWithHttpContextMock`.